pkg/util/kubernetes/kubelet/containers.go

```diff
@@ -113,6 +115,39 @@ func parseContainerInPod(status ContainerStatus, pod *Pod) (*containers.Containe
 	return c, nil
 }
 
+func parseContainerNetworkAdresses(status ContainerStatus, pod *Pod) []containers.NetworkAddress {
+	addrList := []containers.NetworkAddress{}
+	podIP := net.ParseIP(pod.Status.PodIP)
```

Comment:
Do pods in host network mode have a valid `PodIP` field? Is the logic able to handle them?

Reply:
Yes, the `podIP` field is filled with the host IP in host networking mode